daemon/api.go

```diff
@@ -698,7 +698,7 @@ func sideloadSnap(c *Command, r *http.Request) Response {
 	contentType := r.Header.Get("Content-Type")
 
 	if strings.HasPrefix(contentType, "multipart/") {
-		// spec says POSTs to sideload snaps should be “a multipart file upload”
+		// spec says POSTs to sideload snaps should be "a multipart file upload"
```

Comment:
While we can't assume the *terminal* has unicode (and in particular, i think, utf8) support, I think we can assume the developers' editors and browser does. So comments and `.md` files are fine; output of `snap` (and `snappy`) is not.